packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -41,6 +41,10 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     [self deviceSupportsBiometrics:result];
   } else if ([@"isDeviceSupported" isEqualToString:call.method]) {
     result(@YES);
+  } else if ([@"handleAuthReplyWithSuccess" isEqualToString:call.method]) {
+    bool success = [call.arguments[@"success"] boolValue];
+    NSError* error = call.arguments[@"error"];
```

Comment:
is `call.arguments[@"error"]` indeed `NSError` type?

Reply:
It should be!
https://github.com/flutter/plugins/blob/abb8e681c414c6d396ae8ccc0bfc9a1293b1602c/packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m#L215

Reply:
can you double check by setting a break point here and inspecting the type of `error`?

Reply:
<img width="1038" alt="Screen Shot 2022-12-12 at 1 23 04 PM" src="https://user-images.githubusercontent.com/36148254/207157553-b2c6c5c1-2740-4247-b1c6-a4e1310fe119.png">